skimage/measure/_regionprops.py

```diff
@@ -62,10 +64,59 @@
     'WeightedCentralMoments': 'weighted_moments_central',
     'WeightedCentroid': 'weighted_centroid',
     'WeightedHuMoments': 'weighted_moments_hu',
+    'WeightedLocalCentroid': 'weighted_local_centroid',
```

Comment:
I know what happened: this list exists from the old UI to regionprops, which was a dictionary. The missing properties were added after the move to the new UI.

Reply:
@leGIT-bot https://github.com/scikit-image/scikit-image/pull/674 A long time ago! LOL